tests/plugins/webp-uploads/helpers/class-wp-image-edit.php

```diff
@@ -2,7 +2,7 @@
 /**
  * A WP_Image_Edit class to avoid abstraction to handle image edits.
  *
- * @package performance-lab
+ * @package webp-uploads
```

Comment:
See above, the folder with this file should have a consistent name with how you did it for Dominant Color Images.

Reply:
Renamed in https://github.com/WordPress/performance/pull/1011/commits/de7732047571df69397865caf202c80a39331475